examples/belief_propagation/bpexample.jl

```diff
@@ -31,41 +27,33 @@ function main()
   v = (1, 1)
 
   #Now do Simple Belief Propagation to Measure Sz on Site v
-  mts = message_tensors(
-    ψψ; subgraph_vertices=collect(values(group(v -> v[1], vertices(ψψ))))
+  pψψ = PartitionedGraph(ψψ, collect(values(group(v -> v[1], vertices(ψψ)))))
+  mts = belief_propagation(
+    pψψ; contract_kwargs=(; alg="exact"), verbose=true, niters=10, target_precision=1e-3
   )
-
-  mts = belief_propagation(ψψ, mts; contract_kwargs=(; alg="exact"), niters=20)
-
-  numerator_network = approx_network_region(
-    ψψ, mts, [(v, 1)]; verts_tn=ITensorNetwork([apply(op("Sz", s[v]), ψ[v])])
+  numerator_tensors = approx_network_region(
+    pψψ, mts, [(v, 1)]; verts_tensors=[apply(op("Sz", s[v]), ψ[v])]
```

Comment:
```suggestion
    pψψ, mts, [PartitionVertex(v)]; verts_tensors=[apply(op("Sz", s[v]), ψ[v])]
```
Same kind of comment as elsewhere, it seems cleaner and clearer to me to start using the `PartitionVertex` type for this kind of thing, unless I am misunderstanding the code.